### PR TITLE
Add Loki

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

#### What this PR does / why we need it:
For deploy loki.

#### Special notes for your reviewer:
@tomwilkie

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/loki/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| `loki.persistence.storageClassName`                       | Type of persistent volume claim               | `default`                               |
+| `loki.config.auth_enabled`                                | Enable authorization                          | `false`                                 |
+| `loki.config.ingester.lifecycler.ring.store`              | The store way of lifecycler                   | `inmemory`                              |
+| `loki.config.ingester.lifecycler.ring.replication_factor` | The number of lifecycler ring                 | `1`                                     |
+| `loki.schema_configs`                                     | Loki scheam config                            | `[{"from":0,"store":"boltdb","object_store":"filesystem","schema":"v9","index":{"prefix":"index_","period":"168h"}}]` |
+| `loki.storage_configs`                                    | Loki storage config                           | `[{"name":"boltdb","directory":"/data/loki/index"},{"name":"filesystem","directory":"/data/loki/chunks"}]` |
+
+### Promtail
```
Comment (kfox1111) [inline stable/loki/README.md:68]: I'd recommend making promtail/loki either subcharts with a condition making them enable/disablable or wrapping them in conditional logic.

You could easily want to deploy loki on one cluster and have multiple clusters launching promtail and pushing to loki. 
Both should be independently selectable.

File: stable/loki/templates/loki/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      lifecycler:
+        ring:
+          store: {{ .Values.loki.config.ingester.lifecycler.ring.store }}
+          replication_factor: {{ .Values.loki.config.ingester.lifecycler.ring.replication_factor }}
+      chunk_idle_period: 15m
+
+{{- if .Values.loki.config.schema_configs }}
+    schema_config:
+      configs:
+  {{- range .Values.loki.config.schema_configs }}
```
Comment (kfox1111) [inline stable/loki/templates/loki/configmap.yaml:31]: Is there a reason not to just 'toYaml .' inside the range rather then build out each item explicitly?

Comment (daixiang0) [reply stable/loki/templates/loki/configmap.yaml:31]: I saw some code use `toYaml` while some not, i did not know in which case use it or not. So i keep them as they were.

**Comment by daixiang0** [comment]
@mattfarina @prydonius @tomwilkie any input?

**Comment by davkal** [comment]
Hi @daixiang0 we had a discussion with the Loki team and would prefer it if the chart stayed in our repo with the goal of making our repo serve the charts. Also, we're making you an official contributor.
For this PR we preferred it if we closed it.

**Comment by daixiang0** [comment]
Hi, @davkal i get it, close.

**Pull request closed without merging**